node/src/tests.rs

```diff
@@ -264,10 +283,10 @@ pub struct IntegrationTestSetup {
 impl IntegrationTestSetup {
     /// Generates test node configs and a fake indexer; each config can then be used
     /// to start running the node.
-    pub fn new(
+    pub async fn new(
         clock: Clock,
         temp_dir: &Path,
-        participant_accounts: Vec<AccountId>,
+        participant_accounts: Vec<AccountId>, // we create per account. lol.
```

Comment:
Hate to be boring here, but I don't get it, should this be changed?

Reply:
oh yeah, that was a note to myself :D

Reply:
Nice, seems like you had fun coding this :blush: